src/client/public/try.it.ts

```diff
@@ -0,0 +1,38 @@
+import * as $ from 'jquery';
+import '../assets/styles/extras.scss';
+
+interface InitializationParams {
+    origin: string;
+    runnerSnippetUrl: string;
+    wacUrl: string;
+}
+
+(() => {
+    (window as any).in_try_it_mode = true;
+    (window as any).initializeTryIt = initializeTryIt;
+    let snippetId;
+
+    function initializeTryIt(params: InitializationParams): void {
+        $(document).ready(() => {
+            window.addEventListener('message', receiveMessage, false);
```

Comment:
I don't think you need to wait until document.ready to add the "message" event listener, this does not depend on the DOM and it could run into race conditions.